packages/client/hmi-client/src/components/documents/Document.vue

```diff
@@ -227,10 +225,10 @@ const equationArtifacts = computed(
 );
 const urlArtifacts = computed(() =>
 	doc.value?.knownEntities && doc.value.knownEntities.urlExtractions.length > 0
-		? doc.value.knownEntities.urlExtractions
+		? uniqWith(doc.value.knownEntities.urlExtractions, isEqual) // removes duplicate urls
```

Comment:
This is ocmpletely fine. But if you want to go full _Vanilla_ JS you can use [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#remove_duplicate_elements_from_an_array).
```js
[...new Set(doc.value.knownEntities.urlExtractions)]
```